lib/http.js

```diff
@@ -40,10 +39,7 @@ exports.get = function(options, cb) {
 exports._connectionListener = server._connectionListener;
 const Server = exports.Server = server.Server;
 
-exports.createServer = function(requestListener) {
-  return new Server(requestListener);
-};
-
+exports.createServer = requestListener => new Server(requestListener);
```

Comment:
I'd like to see us specifying these sort of things in one clear way (of course, I don't mind aligning with it).